### ISSUE TITLE
Rename master branch to main

### ISSUE THREAD

**Issue by ncoghlan** [jazzband/contextlib2#57]
Pull request queue is clear, so taking the opportunity to rename the master branch to main.

**Comment by ncoghlan** [comment]
@hugovk Actually, it looks like I don't have enough rights on the repo to do that (presumably related to the long ago migration from my personal account to the Jazzband umbrella org). No Settings tab, and attempting to change the branch name just tells me I don't have the required permissions.

**Comment by hugovk** [comment]
Yeah, it needs an admin, see https://github.com/jazzband/help/issues/196.

**Comment by ulgens** [comment]
@ncoghlan Sorry for the edit on your comment. I saw the "add tasklist" button and I thought it's going to create something new, but instead it edited your content. I have no idea how and why I've this permission.
